### PR TITLE
Settings UI update

### PR DESCRIPTION
**Pull request by PixelJanitor**

### CODE REVIEW

File: app/templates/components/organizations/github-settings.hbs [hunk 1]
```diff
@@ -1,50 +1,64 @@
...
-        checked={{if (eq changeset.githubAuthMechanism "github-integration") "checked"}}
-        {{action "changeSelection" preventDefault=false}}
-      >
-      <span class="label-body">GitHub App</span>
-      <p>Shows visual change status in GitHub pull requests. Has customizable access control.</p>
-    </label>
-    {{#if (eq changeset.githubAuthMechanism "github-integration")}}
-      {{organizations/github-integrator organization=organization showSupport="showSupport"}}
-    {{/if}}
+    {{#if (not (await isEligibleForGithubIntegration))}}
```
Comment (maprules1000) [inline app/templates/components/organizations/github-settings.hbs]: This part of the `if` statement is displayed while the data about this integration is loading in the backend. Could we find a way to throw a loading icon up here instead?

File: app/templates/components/forms/organization-user-edit.hbs [hunk 2]
```diff
@@ -8,23 +8,23 @@
         changeset=changeset
         validateProperty=(action "validateProperty" changeset "role")
     }}
-    <p>
+    <p style="margin-top: calc(var(--spacing-1) * -1)" class="mb-2 text-light">
```
[the remark this answers is not in the document]
Comment (PixelJanitor) [reply app/templates/components/forms/organization-user-edit.hbs]: this extra custom style is only needed for this one paragraph tag, so no need to bloat our css with a single class that does one thing for this one element, so I like to keep these abstracted out as a single use inline style.

File: app/templates/components/organizations/billing-section.hbs [hunk 3]
```diff
@@ -49,70 +46,79 @@
...
+                  </div>
+                {{/with}}
+                {{#unless organization.subscription.plan.isTrial}}
+                  <div class="mt-2">
+                    You are on a custom plan, feel free to <a href="#" {{action "showSupport"}}>reach out</a> with any questions before changing plans below.
+                  </div>
+                {{/unless}}
+              </div>
+              <div style="max-width: 150px" class="flex-1 pr-2"></div>
+              <div style="max-width: 150px" class="flex-1 pr-2 flex align-center justify-end">
```
[the remark this answers is not in the document]
Comment (PixelJanitor) [reply app/templates/components/organizations/billing-section.hbs]: same with these

File: app/templates/organizations/organization/users/index.hbs [hunk 4]
```diff
@@ -1,9 +1,16 @@
-<section class="p-2">
-  {{#link-to "organizations.organization.users.invite" model.slug classNames="Button Button--primary u-pull-right"}}
+<section class="px-2 pt-3">
+  {{!-- <h1 class="f3 text-weight-semibold mb-sm">Configure {{capitalize model.name}}</h1> --}}
```
[the remark this answers is not in the document]
Comment (PixelJanitor) [reply app/templates/organizations/organization/users/index.hbs]: Oh snap. Thanks for catching this. This is the ability to add a user and hasn't been styled yet. Would have been a bad thing to push without this functionality back in there :) Styling that now.

**Pull request merged**